lib/reporters/base.js

```diff
@@ -333,6 +340,67 @@ Base.prototype.epilogue = function() {
 };
 
 /**
+ * Opens for writing the file referenced on the `optionName` option
+ * of the reporter options, if any.
+ * Call this method to support writing to this file when calling
+ * write or writeLine.
+ *
+ * @param {string} optionName the name of the option, defaults to 'output'.
+ * @api public
+ */
+Base.prototype.openOutput = function(optionName) {
+  var output = this.reporterOptions && this.reporterOptions[optionName || 'output'];
+  if (output) {
+    if (!fs.createWriteStream) {
+      throw new Error('file output not supported in browser');
+    }
+
+    mkdirp.sync(path.dirname(output));
+    this.fileStream = fs.createWriteStream(output);
```

Comment:
Would it be better just to throw an error instead? 

You shouldn't be calling `openOutput` twice, if you do (and for example you happen to write something in the meantime) that data will be lost. Feels like a path to silent errors for most of the times. If we want to support (for some odd reason) calling it twice, maybe we should add an explicit `closeOutput`?